### DISCUSSION TITLE
How to ignore this field

### DISCUSSION THREAD

**Discussion by jet10000** [jrief/django-formset#21] [Q&A]
<img width="604" alt="image" src="https://user-images.githubusercontent.com/2258120/187879326-eebd982c-f49b-46d6-93e2-a49f9156d873.png">

```
{
  "full_name": "First Last",
  "gender": "f",
  "pregnant": true
}
```

<img width="608" alt="image" src="https://user-images.githubusercontent.com/2258120/187879441-56262732-0996-4e2a-9d24-2b4d0f207e98.png">

```
{
  "full_name": "First Last",
  "gender": "m",
  "pregnant": true
}
```

When i switch to "Male", "pregnant:true" still in submitted data.

I expect the pregnant field to be ignored.

How to ignore this field?

**Comment by jrief** [comment]
The entry `pregnant` is not removed from the submitted payload if `gender != 'female'`, **django-formset** just hides the widget. You may however remove it by override the form's `clean()` method.

Is there any reason, why that could cause a problem for you?
